pkg/network/tracer/tracer_test.go

```diff
@@ -1636,3 +1636,9 @@ func skipIfWindows(t *testing.T) {
 		t.Skip("test unavailable on windows")
 	}
 }
+
+func skipIfNotWindows(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("test only available on windows")
+	}
+}
```

Comment:
Is this included accidentally?

Reply:
yeah.  I originally had a test (that was windows only), but the test didn't actually work so I backed it out.
I'll remove when I update to the signed driver.